schemas/json/theme.json

```diff
@@ -889,6 +889,17 @@
 						}
 					},
 					"additionalProperties": false
+				},
+				"filter": {
+					"description": "Filters to attach to the styles",
+					"type": "object",
+					"properties": {
+						"duotone": {
+							"description": "Sets the duotone filter.",
```

Comment:
What do you think about adding a regex pattern—`^var` or `^var(:preset:duotone|\(--wp--preset--duotone)` or something to that effect? (I'm typing this on my phone, so don't trust those regexs to be 100% correct) 

Technically any CSS filter property will run without issue, but since this is marked as duotone folks should probably be using the generated preset.

Reply:
I'm hesitant to add the regular expression here to make this stricter. I'd rather start off making sure valid themes are validating.

If that's something we want to do in the schema, I think it should be its own change. There are other strings like gradients, palettes, and more that could probably benefit from regexes, if that's the way we want to go.

Reply:
Sounds good to me 👍